rele/config.py

```diff
@@ -102,5 +102,10 @@ def load_subscriptions_from_paths(sub_module_paths, sub_prefix=None, filter_by=N
             if filter_by and not subscription.filter_by:
                 subscription.set_filters(filter_by)
 
-            subscriptions.append(subscription)
-    return subscriptions
+            if subscription.name in subscriptions:
+                raise RuntimeError(
+                    f"Duplicated subscription found: {subscription.name}."
+                )
+
+            subscriptions[subscription.name] = subscription
```

Comment:
Could this be kept as a list and instead test for `subscription.name in subscriptions`?

Reply:
Yeah, but it will iterate through the full list in every iteration to check if the item.name has already been registered. The dict aproach ensures a constant check every validation